docs/configuration.md

```diff
@@ -0,0 +1,696 @@
+   data. It's described in more details [below](#initializer).
+2. Thesholds are specified as floating-point numbers.
+3. `ftsSeed` is base64-encoded, not hex-encoded.
+4. `txFeePolicy` is specified slightly differently. The comments in
+   the example explain the format pretty well. Note that the values
+   are lovelaces, not ADA.
+5. Addresses in `avvmDistr` are presented in base64 format, not base64(url).
+
+#### Initializer
+
+Sample configuration with `initializer` was presented above. It
+contains the following fields:
+* `testBalance`. It specifies how many nodes should be rich, how many
+  nodes should be poor, how big should be total balance of rich nodes,
+  total balance and whether to use HD addresses or simple PubKey
+  addresses. Assignment of balances and stakes is described in
+  detail [below](#balances-and-stakes).
+* `fakeAvvmBalance`. It contains number of fake AVVM seeds and how
+  balance of each seed. These seeds can be used to redeem ADA.
+* `avvmBalanceFactor`. Real AVVM balances will be multiplied by this
+  factor. Fake AVVM balances are not affected.
+* `useHeavyDlg`. If it's `True`, then for each bootstrap stakeholders
+  there will be a heavyweight delegation certificate from this
+  stakeholder to a core node. Otherwise bootstrap stakeholders will
+  participate in the protocol directly.
+* `seed`. It is an integer number which will be used to generate all
+  secrets (and consequenlty all public keys, addresses, etc. which are
+  derived from secret keys). It can be overridden using
+  `--configuration-seed` option.
+
+Initializer makes it quite easy to generate genesis data,
+because all secrets are derived from only one integer. It's suitable
+for custom developer clusters because there we can let some people control all
+stake. It's not true for mainnet, where we have three parties, that's
+why workflow for mainnet is more complex.
+
+Other features supported by initializer:
+* It allows to have fake AVVM seeds to test ADA redemption, which are
+  needed, because developers and QA specialists don't know actual
+  seeds having value.
+* It allows to decrease the value of all AVVM seeds using
+  `avvmBalanceFactor`.
+* It allows to have not only redeem addresses (derived from AVVM
+  seeds), but also ready-to-use public key addresses (specified by
+  `testBalance`). We can have few addresses with very big balance
+  owned by us and many addresses with small balance shared with
+  everybody (e.g. for faucet).
+
+#### Balances and stakes
+
+In _genesis spec_ there are two fields which determine balances and
+stakes: `avvmDistr` and `initializer`. `avvmDistr` is supposed to
+contain the result of vending (which can be found in
+`lib/configuration.yaml`, see `mainnet_avvmDistr`), but it also can be
+empty if you don't need AVVM addresses (or it can be modified in
+a different way, however you want). Overview of `initializer` is
+described [above](#initializer).
+
+In the following we describe how
```

Comment:
In the following ... paragraph i suppose.

Reply:
Apparently this phrase is acceptable too, but I missed the comma https://ell.stackexchange.com/a/94858